functions/src/actions/wayback-machine.js

```diff
@@ -0,0 +1,134 @@
+const request = require('request');
```

Comment:
it is not critical, but I'd recommend to reorganize imports:

1) put apart third part imports and local imports
2) put improts in the alphabet order by name of lib:

```javascript
const mustache = require('mustache');
const traverse = require('traverse');
const xml2js = require('xml2js');
```

You could check such organization in any of my js-files.

Such organization would help to read imports.